pkg/controller/config/config_controller.go

```diff
@@ -7,7 +7,7 @@ import (
 	"path/filepath"
 	"strings"
 	"time"
-
+        runtime1 "runtime"
```

Comment:
🤔

Reply:
@vdemeester  - Hey I while building operator code observed below error hence I named runtime to runtime1. 

# github.com/tektoncd/operator/pkg/controller/config
pkg/controller/config/config_controller.go:107:5: undefined: "k8s.io/apimachinery/pkg/runtime".GOARCH
make: *** [out/operator] Error 2